From 00ca16af1de490690464e53b61cc9a8cb55e884e Mon Sep 17 00:00:00 2001
From: Shawn Sandy <shawnsandy04@gmail.com>
Date: Mon, 14 Apr 2014 13:06:03 -0400
Subject: [PATCH] updates * updated styles sheets * fixed issue(s) with
 e-course quiz styling * corrected some font adjustments * profile view form
 updated now margin-0px

---
 .gitignore                  |  1 +
 webroot/css/theme/style.css | 56 ++++++++++++++++++++++++++++++++++++++++-----
 2 files changed, 51 insertions(+), 6 deletions(-)

diff --git a/.gitignore b/.gitignore
index 737a551..1ab555e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 /img/default/default_header_logo.jpg
 .DS_Store
 css/.DS_Store
+/webroot/css/theme/layout-fix-view.css
\ No newline at end of file
diff --git a/webroot/css/theme/style.css b/webroot/css/theme/style.css
index 1002c1a..193aa75 100755
--- a/webroot/css/theme/style.css
+++ b/webroot/css/theme/style.css
@@ -803,6 +803,7 @@ ul.nav li:first-child {
 /* thumbnails */
 .thumbnails {
   margin-bottom: 0;
+  display: table;
 }
 .thumbnails li {
   margin-bottom: 0px;
@@ -1982,19 +1983,30 @@ span.label a {
 #ecourse-quiz li.question label.main-label {
   display: block;
   font-weight: bold;
+  border-bottom: 1px solid #EEECEC;
 }
 
 #ecourse-quiz li.question .answer { margin-left: 10px }
 
-#ecourse-quiz li.question input[type=radio] { margin-right: 8px }
+#ecourse-quiz li.question input[type=radio] { margin-right: 8px}
 
 #ecourse-quiz li.question input[type=radio],
 #ecourse-quiz li.question input[type=radio] + label {
   margin-top: 10px;
 }
+#ecourse-quiz label.main-label {
+    display: block;
+    width: 100%;
+    padding: 0;
+}
+#ecourse-quiz .required.answer {
+    float: left;
+    padding: 0;
+}
 
 #ecourse-quiz li.question input[type=radio] + label {
   color: #555;
+  margin: 0;
 }
 
 #ecourse-quiz label.error {
@@ -2026,6 +2038,21 @@ span.label a {
   background: url(../../img/theme/icons/exclamation.png) 0 0 no-repeat;
 }
 
+
+p.incorrect-answer {
+    padding: 0;
+    margin: 0;
+}
+
+p.correct-answer {
+    margin: 0;
+}
+
+#ecourse-quiz-results li {
+/*     margin-bottom: 40px; */
+    padding-bottom: 10px;
+}
+
 .pagination {
   margin: 20px 0;
 }
@@ -2400,7 +2427,7 @@ span.label a {
 
  margin-right: auto;
 /*  padding-left:160px; */
- font-size: 16px;
+ font-size: 14px;
  font-family: Arial;
  padding-top: 20px;
  color: #8E8E8E;
@@ -2458,7 +2485,7 @@ padding-right: 40px;
 }
 
 #main-content p {
-  font-size: 16px;
+  font-size: 14px;
   line-height: 1.5;
   font-family: Helvetica, Arial, Verdana, sans-serif;
   font-weight: normal;
@@ -2505,6 +2532,11 @@ padding-right: 40px;
     width: 50%;
     margin: 0;
 }
+
+#content form p.left {
+  margin: 0;
+  padding: 0;
+}
 #content input[type="text"], 
 #content input[type="password"],
 #content input[type="datetime"],
@@ -2524,12 +2556,16 @@ padding-right: 40px;
  width: 50%;
  font-size: 16px;
  min-width: 250px;
-    
 }
-#content select {
+    
+#content select, #content textarea {
     width: 50%;
 }
 
+#contactform .form-group {
+    margin-bottom: 20px;
+}
+
 #content label {
     font-size: 16px;
     margin-bottom: 10px;
@@ -2553,6 +2589,15 @@ padding-right: 40px;
 li.module span.steps.status {
 color: #484F57 !important;
 }
+//etabs
+ul.etabs{} 
+.etabs { margin: 0; padding: 0; }
+.tab { display: inline-block; zoom:1; *display:inline; background: #eee; border: solid 1px #999; border-bottom: none; -moz-border-radius: 4px 4px 0 0; -webkit-border-radius: 4px 4px 0 0; }
+.tab a { font-size: 12px; line-height: 1.5em; display: block; padding: 0 10px; outline: none; }
+.tab a:hover { text-decoration: underline; }
+.tab.active { background: #fff; padding-top: 6px; position: relative; top: 1px; border-color: #666; }
+.tab a.active { font-weight: bold; }
+.tab-container .panel-container { background: #fff; border: solid #666 1px; padding: 10px; -moz-border-radius: 0 4px 4px 4px; -webkit-border-radius: 0 4px 4px 4px; }
 
 /* browser issue fix resize to 1180 */
 @media (max-width: 1201px) {
@@ -2591,4 +2636,3 @@ color: #484F57 !important;
     padding-top: 60px;
 }
 
-
-- 
1.8.4.2

